update.sh

```diff
@@ -1,42 +1,60 @@
 #!/bin/bash
 
-set -eu
+set -eux
 
-if [ $# != 3 ]; then
-  echo "Usage: $0 MYSQL_VERSION MROONGA_VERSION GROONGA_VERSION"
-  echo " e.g.: $0 8.0.30 12.06 12.0.6"
+if [ $# != 0 ]; then
+  echo "Usage: $0"
   exit 1
 fi
 
-mysql_version=$1
-mroonga_version=$2
-groonga_version=$3
+target_mysqls=(
```

Comment:
```suggestion
target_mysql_versions=(
```

Reply:
Sorry this is right. Please ignore my comment.

Reply:
Oh...